airflow/contrib/operators/dataproc_operator.py

```diff
@@ -272,7 +307,7 @@ def _wait_for_done(self, service):
                 time.sleep(15)
 
     def _get_init_action_timeout(self):
-        match = re.match(r"^(\d+)(s|m)$", self.init_action_timeout)
+        match = re.match(r"^(\d+)([sm])$", self.init_action_timeout)
```

Comment:
just double check, should the regex pattern be [smdh] as well to be consistent with the get_graceful_decommission_timeout?

Reply:
Possibly?  That would require some other substantive changes right below this line to handle the `d` and `h`.  My change was just to make the regex a little more standardized, but sometimes those can spill over to more substantive changes.  I would want to err on the side of making that a separate change.